pkg/client/ethereum.go

```diff
@@ -185,17 +185,6 @@ func (c *HTTPClient) ECRecover(ctx context.Context, storeName string, req *types
 	return parseStringResponse(response)
 }
 
-func (c *HTTPClient) Verify(ctx context.Context, storeName string, req *types.VerifyRequest) error {
```

Comment:
What is the reason to delete this?

Reply:
Verify endpoint was removed but we forgot to remove it from the client.